bin/accessibility-automation/plugin/index.js

```diff
@@ -0,0 +1,47 @@
+
+const path = require("node:path");
+
+const browserstackAccessibility = (on, config) => {
+  let browser_validation = true;
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    try {
+
+      if (browser.name !== 'chrome') {
+        console.log(`Accessibility Automation will run only on Chrome browsers.`);
+        browser_validation = false;
+      }
+      if (browser.majorVersion <= 94) {
+        console.log(`Accessibility Automation will run only on Chrome browser version greater than 94.`);
```

Comment:
This would also get printed for non chrome browsers

Reply:
fixed